api/src/main/java/com/api/demo/models/ReportReview.java

```diff
@@ -0,0 +1,56 @@
+package com.api.demo.models;
+
+import javax.persistence.*;
+import java.sql.Date;
+
+
+@Entity
+public class ReportReview {
+
+    @Id
+    @GeneratedValue( strategy= GenerationType.AUTO )
+
+    private int reportId;
+
+    @OneToOne
+    private User author;
```

Comment:
se calhar, como é one to one, conseguiremos generalizar o report  para dar logo para o jogo e user ao mesmo tempo